src/cljs/rems/administration/create_form.cljs

```diff
@@ -81,12 +81,13 @@
 (rf/reg-event-db ::set-form-field (fn [db [_ keys value]] (assoc-in db (concat [::form] keys) value)))
 
 (rf/reg-event-db
- ::add-form-field
- (fn [db [_]]
-   (let [stable-id (generate-stable-id)]
-     (focus-field-editor! stable-id)
-     (update-in db [::form :form/fields] items/add {:field/stable-id stable-id
-                                                    :field/type :text}))))
+  ::add-form-field
```

Comment:
I'll send you code style settings for this project, since at default settings the indentation is different.